pkg/rancher-desktop/components/MarketplaceCard.vue

```diff
@@ -76,12 +178,22 @@ export default {
         font-size: 0.8rem;
         font-weight: 400;
       }
-    }
 
-    img {
-      width: 50px;
-      height: 50px;
+      img {
+        max-width: 40px;
+        width: 100%;
+        max-height: 40px;
+      }
     }
   }
+
+  &-card-footer {
+    align-content: center;
+    display: flex;
+    align-content: flex-start;
+    margin-top: 15px;
+    place-items: center;
+    width: 100%;
+  }
```

Comment:
Are these styles from a previous implementation that might have had more content? I'm not seeing what a lot of these card footer styles are doing (with the exception of `margin-top`).